addons/block_code/ui/block_canvas/block_canvas.tscn

```diff
@@ -60,12 +60,34 @@ theme_override_constants/margin_top = 4
 theme_override_constants/margin_right = 4
 theme_override_constants/margin_bottom = 4
 
-[node name="ZoomButton" type="Button" parent="WindowContainer/Overlay/MarginContainer"]
+[node name="ZoomButtons" type="HBoxContainer" parent="WindowContainer/Overlay/MarginContainer"]
 unique_name_in_owner = true
+visible = false
+layout_mode = 2
+
+[node name="ZoomOutButton" type="Button" parent="WindowContainer/Overlay/MarginContainer/ZoomButtons"]
+modulate = Color(1, 1, 1, 0.470588)
+custom_minimum_size = Vector2(25, 0)
+layout_mode = 2
+focus_mode = 0
+theme_override_font_sizes/font_size = 24
+text = "-"
```

Comment:
This should be an icon button. The Godot editor has icons for zoom less / more:
- https://github.com/godotengine/godot/blob/master/editor/icons/ZoomLess.svg
- https://github.com/godotengine/godot/blob/master/editor/icons/ZoomMore.svg